proxy/src/transport/tls/config.rs

```diff
@@ -73,19 +73,33 @@ impl CommonSettings {
             self.end_entity_cert.clone(),
             self.private_key.clone(),
         ];
+
+        fn last_modified(path: &PathBuf) -> Option<SystemTime> {
+            // We have to canonicalize the path _every_ time we poll the fs,
+            // rather than once when we start watching, because if it's a
+            // symlink, the target may change. If that happened, and we
+            // continued watching the original canonical path, we wouldn't see
+            // any subsequent changes to the new symlink target.
+            path.canonicalize()
+                .and_then(|canonical| {
+                    trace!("last_modified: {:?} -> {:?}", path, canonical);
```

Comment:
This comment is not going to be useful, especially because it doesn't include the last modified time.

Reply:
WDYM?

Reply:
Ah, you meant the log and not the comment. I found this useful while debugging, as it shows the canonical path that the symlink expanded to. If the file changed, we log the last modified time [on line 105](https://github.com/runconduit/conduit/pull/1076/files/c8e74574baf42e7f1770f18b6dd3bcdd077e2ad6#diff-a4b5d277de7db26e1d308626fdad076aR105) still.

Reply:
yeah, i guess my point is this is the sort of thing that is only useful when you're working on it. not a huge problem either way.

Reply:
Oh, okay. I can open another PR to remove it.

Reply:
probably not worth it, we can clean it up later